src/content/docs/release-notes/docs-release-notes/docs-9-20-2024.mdx

```diff
@@ -0,0 +1,47 @@
+---
+subject: "Docs"
+releaseDate: '2024-09-20'
+version: 'version: September 13-19, 2024'
+---
+
+### New docs
+
+* [IAST configuration](/docs/iast/iast-configuration) explains how to configure and schedule IAST scans.
+* [OCSF integration] describes how to use this new integration to monitor your Open Cybersecurity Schema Framework data.
+
+### Minor changes
+
+* Updated the [requirements and compatibility](/install/vsphere/#requirements-and-compatibility) information for [vSphere monitoring integration](/install/vsphere). Also, highlighted information about the workaround to report all data if your environment consists of more than 800 virtual machines.
+* Updated the code snippet required to [forward Envoy logs to New Relic](/docs/infrastructure/host-integrations/host-integrations-list/envoy-integration/#forward-envoy-logs).
+* Updated the code snippets required to [configure NRI-Flex for Nextcloud](/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration/#integrate) and to [forward Nextcloud logs to New Relic](/docs/infrastructure/host-integrations/host-integrations-list/nextcloud-integration/#forward-nextcloud-logs-to-new-relic).
+* Updated [.NET agent compatibility and requirements](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements) for:
+    * `StackExchange.Redis` (both minimum supported version and latest verified compatible version)
+    * `AWS Bedrock` (latest verified compatible version)
+    * `Confluent.Kafka` (latest verified compatible version)
+    * `AWSSDK.SQS` (both minimum supported version and latest verified compatible version)
+* Updated the [Node.js agent compatibility information](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent) for `@aws-sdk/client-bedrock-runtime`, `@aws-sdk/client-dynamodb`, `@aws-sdk/client-sns`, `@aws-sdk/client-sqs`, `@aws-sdk/lib-dynamodb`, `@aws-sdk/smithy-client`, `@langchain/core`, `@smithy/smithy-client`, `express`, `fastify`, `mysql2`, `next`, and `openai`.
+* Updated the instructions on how to [see your control plane data](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring/#check-integration-works) in [configure control plane monitoring](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring).
+
+### Release notes and What's New posts
+
+* What's New post for:
```

Comment:
```suggestion
Check out our What's New posts to learn about new features and releases:
```

Reply:
I took out the bullet point since I don't see release notes, just itemized what's new. :)

Reply:
Good call!